### PR TITLE
169 double check test coverage to be 75

### PR DESCRIPTION
**Pull request by serainaschraff**
Test coverage is above 75% overall and in every important class

### CODE REVIEW

File: src/main/java/ch/uzh/ifi/hase/soprafs22/service/GameService.java [hunk 1]
```diff
@@ -247,7 +247,7 @@ public Game getGameFromRandomUser(User user) {
     return somePage.getContent().get(0);
   }
 
-
+//TODO: saveGame has no usage.. can it be deleted?
     public void saveGame(Game activeGame) {
       gameRepository.saveAndFlush(activeGame);
     }
```
Comment (anhuwi) [inline src/main/java/ch/uzh/ifi/hase/soprafs22/service/GameService.java:253]: strange, that no usage. if no usage: sure. where do we saveandflush game?

File: src/test/java/ch/uzh/ifi/hase/soprafs22/controller/GameControllerTest.java [hunk 2]
```diff
@@ -153,23 +153,22 @@ void givenUser_createGame() throws Exception {
...
 
         // then
         mockMvc.perform(getRequest).andExpect(status().isOk())
                 .andExpect(jsonPath("$.gameId",is(111)))
                 .andExpect(jsonPath("$.blackCard.id",is(3)))
                 .andExpect(jsonPath("$.blackCard.text",is(blackCard.getText())));
-    }*/
+    }
 
     @Test
```
Comment (anhuwi) [inline src/test/java/ch/uzh/ifi/hase/soprafs22/controller/GameControllerTest.java:173]: niice:)

File: src/test/java/ch/uzh/ifi/hase/soprafs22/controller/UserControllerTest.java [hunk 3]
```diff
@@ -528,12 +465,8 @@ void test_unmatch_user_success() throws Exception {
...
-    User otherUser = new User();
-
-    user.setId(1L);
-    user.setToken("token");
-    otherUser.setId(2L);
+    User user = fillUser(1L, "token");
+    User otherUser = fillUser(2L, "token2");
 
     //given(userService.deleteMatchBetweenUsers(Mockito.anyLong(), Mockito.anyLong())).willThrow(new ResponseStatusException(HttpStatus.NOT_FOUND));
     Mockito.doThrow(new ResponseStatusException(HttpStatus.NOT_FOUND)).when(userService).deleteMatchBetweenUsers(Mockito.anyLong(), Mockito.anyLong());
```
Comment (anhuwi) [inline src/test/java/ch/uzh/ifi/hase/soprafs22/controller/UserControllerTest.java:472]: delete outcomented code?

File: src/test/java/ch/uzh/ifi/hase/soprafs22/controller/UserControllerTest.java [hunk 4]
```diff
@@ -563,12 +492,8 @@ void test_block_user_success() throws Exception {
...
```
Comment (anhuwi) [inline src/test/java/ch/uzh/ifi/hase/soprafs22/controller/UserControllerTest.java:499]: delete outcomented code

File: src/main/java/ch/uzh/ifi/hase/soprafs22/service/GameService.java (hunk 1 above)
Comment (anhuwi) [reply src/main/java/ch/uzh/ifi/hase/soprafs22/service/GameService.java:253]: in createGame. you can delete this!

File: src/test/java/ch/uzh/ifi/hase/soprafs22/controller/UserControllerTest.java (hunk 4 above)
Comment (serainaschraff) [reply src/test/java/ch/uzh/ifi/hase/soprafs22/controller/UserControllerTest.java:495]: done

File: src/test/java/ch/uzh/ifi/hase/soprafs22/controller/UserControllerTest.java (hunk 3 above)
Comment (serainaschraff) [reply src/test/java/ch/uzh/ifi/hase/soprafs22/controller/UserControllerTest.java:469]: done

File: src/main/java/ch/uzh/ifi/hase/soprafs22/service/GameService.java (hunk 1 above)
Comment (serainaschraff) [reply src/main/java/ch/uzh/ifi/hase/soprafs22/service/GameService.java]: done

**Pull request merged**